src/overlays/actors/ovl_Boss_02/z_boss_02.c

```diff
@@ -1932,113 +1998,121 @@ void func_809DD934(Boss02* this, PlayState* play) {
 
-                    temp_a0_5->floorHeight -= 3150.0f;
-                    temp_a0_5->floorHeight *= 10.0f;
+                    actorIter->floorHeight -= 3150.0f;
+                    actorIter->floorHeight *= 10.0f;
 
-                    phi_f0_2 = 10.0f;
+                    scale = 10.0f;
                 } else {
-                    temp_a0_5->world.pos.y *= 0.1f;
-                    temp_a0_5->world.pos.y += 3150.0f;
+                    actorIter->world.pos.y *= 0.1f;
+                    actorIter->world.pos.y += 3150.0f;
 
-                    temp_a0_5->floorHeight *= 0.1f;
-                    temp_a0_5->floorHeight += 3150.0f;
+                    actorIter->floorHeight *= 0.1f;
+                    actorIter->floorHeight += 3150.0f;
 
-                    phi_f0_2 = 0.1f;
+                    scale = 0.1f;
                 }
 
-                temp_a0_5->world.pos.x *= phi_f0_2;
-                temp_a0_5->world.pos.z *= phi_f0_2;
+                actorIter->world.pos.x *= scale;
+                actorIter->world.pos.z *= scale;
 
-                temp_a0_5->speed *= phi_f0_2;
+                actorIter->speed *= scale;
 
-                temp_a0_5->velocity.x *= phi_f0_2;
-                temp_a0_5->velocity.y *= phi_f0_2;
-                temp_a0_5->velocity.z *= phi_f0_2;
+                actorIter->velocity.x *= scale;
+                actorIter->velocity.y *= scale;
+                actorIter->velocity.z *= scale;
 
-                temp_a0_5->gravity *= phi_f0_2;
-                temp_a0_5->terminalVelocity *= phi_f0_2;
+                actorIter->gravity *= scale;
+                actorIter->terminalVelocity *= scale;
 
-                temp_a0_5->scale.x *= phi_f0_2;
-                temp_a0_5->scale.y *= phi_f0_2;
-                temp_a0_5->scale.z *= phi_f0_2;
+                actorIter->scale.x *= scale;
+                actorIter->scale.y *= scale;
+                actorIter->scale.z *= scale;
 
-                if (temp_a0_5->id == ACTOR_ITEM_B_HEART) {
-                    ((ItemBHeart*)temp_a0_5)->baseScale *= phi_f0_2;
+                if (actorIter->id == ACTOR_ITEM_B_HEART) {
+                    ((ItemBHeart*)actorIter)->baseScale *= scale;
                 }
             }
-            temp_a0_5 = temp_a0_5->next;
+
+            actorIter = actorIter->next;
         }
     }
 
-    switch (this->unk_1D78) {
-        case 0:
+    switch (this->giantsMaskCsFlashState) {
+        case GIANTS_MASK_CS_FLASH_STATE_NOT_STARTED:
             break;
 
-        case 1:
-            this->unk_1D7A = 0;
-            func_809DA1D0(play, 255, 255, 255, 0);
-            this->unk_1D78 = 2;
+        case GIANTS_MASK_CS_FLASH_STATE_STARTED:
+            this->giantsMaskCsFlashAlpha = 0;
+            Boss02_FillScreen(play, 255, 255, 255, 0);
+            this->giantsMaskCsFlashState = GIANTS_MASK_CS_FLASH_STATE_INCREASE_ALPHA;
             Audio_PlaySfx(NA_SE_SY_TRANSFORM_MASK_FLASH);
+            // fallthrough
 
```

Comment:
```suggestion
            // fallthrough
```
No newline between fallthroughs and new cases.